### PR TITLE
Setup docker-compose for local development

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: Dockerfile-dev [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+FROM elixir:1.7.3
+
+# Install debian packages
+RUN apt-get update
+RUN apt-get install --yes build-essential inotify-tools postgresql-client
+RUN echo "nameserver 8.8.8.8" >> /etc/resolv.conf
```
Comment (valo) [inline Dockerfile-dev]: Why is this needed?

Comment (IvanIvanoff) [reply Dockerfile-dev]: The `mix deps.get` couldn't resolve the hex links for some reason, though now I tested and it seems to work without this change

File: README.md [hunk 2]
```diff
@@ -36,11 +36,23 @@ This setup is going to start 2 processes:
 If you have docker you can run the app simply by running:
 
 ```bash
-$ docker-compose up
+$ bin/start.sh
```
Comment (IvanIvanoff) [inline README.md]: @valo There is no such file, is it not pushed or is a typo?

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
+      dockerfile: Dockerfile-dev
+
+    volumes:
+      - ./:/app:delegated
+
+    hostname: sanbase_host
+
+    command: sh -c "elixir --sname sanbase --cookie sanbase -S mix phx.server"
+    expose:
+      - 3000
```
Comment (tspenov) [inline docker-compose.yaml]: I think 3000 is not needed now. We should run the frontend service there at some point though.

File: README.md [hunk 4]
```diff
@@ -36,11 +36,23 @@ This setup is going to start 2 processes:
...
 ```
 
 This is going to run the app on port 4000, so [`localhost:4000`](http://localhost:4000) should be accessible.
 
+There will no data in the database, so run this command to fill some information in the database:
```
Comment (tspenov) [inline README.md]: type: `There will be no data`

File: docker-compose.yaml [hunk 5]
```diff
@@ -1,39 +1,46 @@
...
+
+    depends_on:
+      - postgres
       - influxdb
-      - faktory
+
     environment:
-      PORT: 4000
-      DATABASE_URL: postgres://postgres:password@db:5432/sanbase-db
+      DATABASE_URL: postgres://postgres:postgres@postgres:5432/santiment
```
Comment (tspenov) [inline docker-compose.yaml]: I guess here should be `0.0.0.0` for host instead of `postgres` unless you have it in `/etc/hosts` ?

File: docker-compose.yaml [hunk 6]
```diff
@@ -1,39 +1,46 @@
-version: '2'
+version: '3.4'
 services:
-  db:
-    image: postgres
+  postgres:
+    image: timescale/timescaledb:0.10.1-pg10
     ports:
-      - "5432"
+      - "5432:5432"
```
Comment (tspenov) [inline docker-compose.yaml]: `6432:5432` -> so we don't have to stop local postgres ?

**Comment by tspenov** [comment]
@valo  @IvanIvanoff  works for me locally

**Comment by valo** [comment]
I think are missing a script for running the tests. I'll add that
